R/zchunk_LB109.ag_an_ALL_R_C_Y.R

```diff
@@ -33,61 +33,209 @@ module_aglu_LB109.ag_an_ALL_R_C_Y_DISABLED <- function(command, ...) {
+      mutate(year = as.integer(year)) # year is character, convert to integer
+
+    L103.ag_Prod_Mt_R_C_Y <- get_data(all_data, "L103.ag_Prod_Mt_R_C_Y") %>%
+      mutate(year = as.integer(year)) # convert to integer
+
+    L105.an_Food_Mt_R_C_Y <- get_data(all_data, "temp-data-inject/L105.an_Food_Mt_R_C_Y") %>%
+      # The following two lines of code will be removed later, when we're using 'real' data
+      gather(year, value, -GCAM_region_ID, -GCAM_commodity) %>%   # reshape
+      mutate(year = as.integer(substr(year, 2, 5)))   # change Xyear to year
+
+    L105.an_Prod_Mt_R_C_Y <- get_data(all_data, "temp-data-inject/L105.an_Prod_Mt_R_C_Y") %>%
+      # The following two lines of code will be removed later, when we're using 'real' data
+      gather(year, value, -GCAM_region_ID, -GCAM_commodity) %>%   # reshape
+      mutate(year = as.integer(substr(year, 2, 5)))   # change Xyear to year
+
+    L106.ag_NetExp_Mt_R_C_Y <- get_data(all_data, "temp-data-inject/L106.ag_NetExp_Mt_R_C_Y") %>%
+      # The following two lines of code will be removed later, when we're using 'real' data
+      gather(year, value, -GCAM_region_ID, -GCAM_commodity) %>%   # reshape
+      mutate(year = as.integer(substr(year, 2, 5)))   # change Xyear to year
+
+    L106.an_NetExp_Mt_R_C_Y <- get_data(all_data, "temp-data-inject/L106.an_NetExp_Mt_R_C_Y") %>%
+      # The following two lines of code will be removed later, when we're using 'real' data
+      gather(year, value, -GCAM_region_ID, -GCAM_commodity) %>%   # reshape
+      mutate(year = as.integer(substr(year, 2, 5)))   # change Xyear to year
+
+    L108.ag_Feed_Mt_R_C_Y <- get_data(all_data, "temp-data-inject/L108.ag_Feed_Mt_R_C_Y") %>%
+      # The following two lines of code will be removed later, when we're using 'real' data
+      gather(year, value, -GCAM_region_ID, -GCAM_commodity) %>%   # reshape
+      mutate(year = as.integer(substr(year, 2, 5)))   # change Xyear to year
+
+    L108.ag_NetExp_Mt_R_FodderHerb_Y <- get_data(all_data, "temp-data-inject/L108.ag_NetExp_Mt_R_FodderHerb_Y") %>%
+      # The following two lines of code will be removed later, when we're using 'real' data
+      gather(year, value, -GCAM_region_ID, -GCAM_commodity) %>%   # reshape
+      mutate(year = as.integer(substr(year, 2, 5)))   # change Xyear to year
+
+    L122.in_Mt_R_C_Yh <- get_data(all_data, "temp-data-inject/L122.in_Mt_R_C_Yh") %>%
+      # The following two lines of code will be removed later, when we're using 'real' data
+      gather(year, value, -GCAM_region_ID, -GCAM_commodity) %>%   # reshape
+      mutate(year = as.integer(substr(year, 2, 5)))   # change Xyear to year
+
+    # Part 1: Primary agricultural goods
+    # List of all flows for primary agricultural good balances
+    ag_Flow_cols <- c( "Prod_Mt", "NetExp_Mt", "Supply_Mt", "Food_Mt", "Feed_Mt", "Biofuels_Mt", "OtherUses_Mt" )
+    # List of any commodities (e.g. pasture, residue, scavenging) in feed but not in production tables
```

Comment:
This comment seems wrong. Isn't this code getting the names of the commodities that are in the production table?